packages/office-ui-fabric-react/src/components/Icon/FontIcon.tsx

```diff
@@ -25,7 +31,11 @@ export const getIconContent = memoizeFunction((iconName?: string) => {
  */
 export const FontIcon: React.FunctionComponent<IFontIconProps> = props => {
   const { iconName, className, style = {} } = props;
-  const { iconClassName, children, fontFamily } = getIconContent(iconName);
+  const { iconClassName, children, fontFamily } = getIconContent(iconName) || {
+    iconClassName: undefined,
```

Comment:
You shouldn't need to specify all of these, you can just do `getIconContent(iconName) || {}`. (If the compiler complains about that, we can add an interface for the getIconContent return type.)

Reply:
Added a new interface IIconContent as simply doing getIconContent(iconName) || {} was giving compiler error.